src/helpers/index.ts

```diff
@@ -126,7 +121,7 @@ async function fixSaveFolder(
 ) {
   const { user, account_id: epicId } = await ipcRenderer.invoke('getUserInfo')
   const username = isProton ? 'steamuser' : user
-  const isWin = process.platform === 'win32';
+  const isWin = await getPlatform() === 'win32';
```

Comment:
I think this will return a promise and not the platform.
can you test it, otherwise it will break the save-sync functionality,

Reply:
I'll check it out

Reply:
fixed